docs/func.md

```diff
@@ -0,0 +1,578 @@
+# Function Reference
+
+*TODO: compose, flip, search-sorted, aref*
+
+## `debug-print`
+
+Prints the supplied value(s).
+
+```Lisp
+elps> (set 'test (sorted-map "K" "V"))
+(sorted-map "K" "V")
+elps> (debug-print "Hello" test)
+"Hello" (sorted-map "K" "V")
+()
+```
+
+## `json:dump-string`
+
+JSON encodes a value, returning the JSON as a string.
+
+```Lisp
+elps> (json:dump-string (sorted-map "K" "V"))
+"{\"K\":\"V\"}"
+```
+
+## `json:dump-bytes`
+
+JSON encodes a value, returning the JSON as bytes.
+
+```Lisp
+elps> (json:dump-bytes (sorted-map "K" "V"))
+#<bytes 123 34 75 34 58 34 86 34 125>
+elps> (to-string (json:dump-bytes (sorted-map "K" "V")))
+"{\"K\":\"V\"}"
+```
+
+## `to-string`
+
+Converts primitive values to their string representation.
+
+```Lisp
+elps> (to-string 1.0)
+"1"
+elps> (to-string 1.01)
+"1.01"
+elps> (to-string true)
+"true"
+```
+
+## `to-bytes`
+
+Converts a string value to bytes.
+
+```Lisp
+elps> (to-bytes "ABC123")
+#<bytes 65 66 67 49 50 51>
+```
+
+## `to-int`
+
+Converts string digits ([0-9]+) and floats to integers. For floats the mantissa
```

Comment:
I don't think "mantissa" is the correct word to use. I would say "fractional part" or "fractional component."